samza-api/src/main/java/org/apache/samza/operators/data/IncomingSystemMessage.java

```diff
@@ -23,54 +23,47 @@
 
 
 /**
- * This class implements a {@link Message} that encapsulates an {@link IncomingMessageEnvelope} from the system
- *
+ * A {@link Message} implementation that provides additional information about its input {@link SystemStreamPartition}
+ * and its {@link Offset} within it.
```

Comment:
Ah, It was not super-obvious to me that the `it` was referring to the `SSP`. 
+1 if the intention was to point out that offsets are comparable within SSPs.